src/components/footer.js

```diff
@@ -59,6 +59,24 @@ const Footer = () => {
               Releases
             </a>
           </div>
+          <div>
+            <a
+              href="https://github.com/rahuldkjain/github-profile-readme-generator/issues"
+              aria-label="Releases on Github rahuldkjain/github-profile-readme-generator"
+              target="blank"
+            >
+              Issues
+            </a>
+          </div>
+          <div>
+            <a
+              href="https://github.com/rahuldkjain/github-profile-readme-generator/pulls"
+              aria-label="Releases on Github rahuldkjain/github-profile-readme-generator"
```

Comment:
change aria-label to `Pull Requests in rahuldkjain/github-profile-readme-generator`